src/Carnac.Logic/packages.config

```diff
@@ -1,13 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Fody" version="1.26.4" targetFramework="net40" developmentDependency="true" />
-  <package id="Ix-Main" version="1.2.3" targetFramework="net40" />
-  <package id="PropertyChanged.Fody" version="1.49.0" targetFramework="net40" developmentDependency="true" />
-  <package id="Rx-Core" version="2.2.5" targetFramework="net40" />
-  <package id="Rx-Interfaces" version="2.2.5" targetFramework="net40" />
-  <package id="Rx-Linq" version="2.2.5" targetFramework="net40" />
-  <package id="Rx-Main" version="2.2.5" targetFramework="net40" />
-  <package id="Rx-PlatformServices" version="2.2.5" targetFramework="net40" />
+  <package id="Fody" version="1.29.4" targetFramework="net40" developmentDependency="true" />
+  <package id="PropertyChanged.Fody" version="1.52.1" targetFramework="net40" developmentDependency="true" />
   <package id="SettingsProviderNet" version="2.1.1" targetFramework="net40" />
-  <package id="YamlDotNet" version="3.5.1" targetFramework="net40" />
+  <package id="System.Interactive" version="3.1.1" targetFramework="net45" />
+  <package id="System.Reactive" version="3.1.1" targetFramework="net45" />
+  <package id="System.Reactive.Core" version="3.1.1" targetFramework="net45" />
+  <package id="System.Reactive.Interfaces" version="3.1.1" targetFramework="net45" />
+  <package id="System.Reactive.Linq" version="3.1.1" targetFramework="net45" />
+  <package id="System.Reactive.PlatformServices" version="3.1.1" targetFramework="net45" />
+  <package id="System.Reactive.Windows.Threading" version="3.1.1" targetFramework="net45" />
```

Comment:
Damn, you're right. Apologies.

<img width="756" src="https://cloud.githubusercontent.com/assets/359239/25069269/7d671b9e-22bf-11e7-98a8-7fb24ba68f90.png">